runtime/onert/core/src/backend/BackendContext.cc

```diff
@@ -57,7 +58,10 @@ void BackendContext::initConsts()
     const auto &obj = _graph->operands().at(ind);
     if (obj.isConstant() && !constant_initializer->exist(ind))
     {
-      constant_initializer->registerPermuteInitializer(ind, obj);
+      if (_backend->config()->id() == "cpu")
+        constant_initializer->registerExternalInitializer(ind, obj);
+      else
+        constant_initializer->registerPermuteInitializer(ind, obj);
```

Comment:
I hope we could avoid this way.

Reply:
Yes, it looks ugly. I'll try avoiding this way.